packages/app/src/App.js

```diff
@@ -1,24 +1,32 @@
-import React from "react";
-import GoogleBtn from "./components/GoogleBtn/GoogleBtn";
-import "./App.css";
-import FamilyMemberInfo from "./components/familyInformation/familyInformation";
-import family_info from "./components/familyInformation/dummyData";
-import {Route, BrowserRouter, Switch} from 'react-router-dom';
+import React from 'react';
+import { Route, BrowserRouter, Switch } from 'react-router-dom';
+import './App.css';
+import FamilyMemberInfo from './components/familyInformation/familyInformation';
+import family_info from './components/familyInformation/dummyData';
+import SearchBar from './components/SearchBar';
+import Login from './components/pages/Login';
+import { NominationsDataProvider } from './utils/context/NominationsContext';
 
 function App() {
   return (
     <BrowserRouter>
       <Switch>
-        <Route path='/applicationdetails'>
-          <div style={{ margin: "0 auto", backgroundColor: 'grey', padding: '2em'}}>
-            <FamilyMemberInfo {...family_info} />
-          </div>
-        </Route>
-        <Route path='/'>
-          <div className='App'>
-            <GoogleBtn />
-          </div>
-        </Route>
+        <NominationsDataProvider>
+          <Route path="/applicationdetails">
+            <div
+              style={{
+                margin: '0 auto',
+                backgroundColor: 'grey',
+                padding: '2em',
+              }}
+            >
+              <FamilyMemberInfo {...family_info} />
+            </div>
+          </Route>
+          <Route exact path={'/login'} component={Login} />
+
+          <SearchBar />
```

Comment:
It feels a little weird to me to have the `SearchBar` component here outside the router components. Would it make more sense to just have it replace the `GoogleBtn` component in the root route?